src/request/components/RequestDetailPanelRequest.tsx

```diff
@@ -79,4 +90,64 @@ export class Request extends React.Component<IRequestProps, {}> {
             </div>
         );
     }
+
+    private renderMiddleware() {
+        return (
+            <div className='tab-request-middleware'>
```

Comment:
No, TypeScript leans towards single quotes (`'`) as string delimiters, and we have linting rules to enforce that (at least, we do in other repos).